### PR TITLE
Company basic profile info template

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-136)

Desktop:
desktop

mobile:
mobile

### CODE REVIEW

**Comment by konradko** [comment]
@richtier rebase on master plz

**Comment by richtier** [comment]
@konradko @monikasulik

File: registration/templates/company-profile-details.html [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
+			<div class="span8 company-profile-box">
+				<h3>Company description</h3>
+				<p>{{company.description|default:'Please set your company description.'}}</p>
+			</div>
+		</div>
+		<div class="row-fluid"></div>
+	</div>
+</div>
+{% endblock %}
+>>>>>>> Add company profile details template
```
Comment (monikasulik) [inline registration/templates/company-profile-details.html]: :umbrella:

File: registration/templates/company-profile-details.html [hunk 2]
```diff
@@ -1,5 +1,51 @@
...
+		<div class="row-fluid">
+			<div class="span4 company-profile-box">
+				<h3>Company number</h3>
+				<p>{{company.number}}</p>
+			</div>
+			<div class="span8 company-profile-box">
+				<h3>My export aims</h3>
+				<p>
+					<ul>
+						{% for aim in company.aims %}
```
Comment (monikasulik) [inline registration/templates/company-profile-details.html]: I think you probably need to stub the aims in the view as well - currently not there.

File: registration/templates/company-profile-details.html [hunk 3]
```diff
@@ -1,5 +1,51 @@
...
+				<div class="span1 company-profile-details-hero-edit-button">
+					<a class="btn btn-primary" href="{% url 'company-edit' %}">Edit</a>
+				</div>
+				<div class="clearfix"></div>
+			</div>
+		</div>
+		<div class="row-fluid">
+			<div class="span4 company-profile-box">
+				<h3>Company number</h3>
+				<p>{{company.number}}</p>
```
Comment (monikasulik) [inline registration/templates/company-profile-details.html]: Needs stubbing in the view

File: registration/tests/test_templates.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+
+def test_company_profile_details_renders_description():
+    html = render_to_string('company-profile-details.html', default_context)
+    assert 'Exporters of UK wares.' in html
+
+
+def test_company_profile_details_renders_website():
+    html = render_to_string('company-profile-details.html', default_context)
+    assert 'Exporters of UK wares.' in html
```
Comment (monikasulik) [inline registration/tests/test_templates.py]: This needs to be a website :spider_web:

**Comment by monikasulik** [comment]
I haven't looked at the css/styling stuff (I'm totally not the person for that kind of stuff :)), so you might want @franciscobenedict to take a quick look as well?

File: registration/templates/company-profile-details.html (hunk 3 above)
Comment (richtier) [inline registration/templates/company-profile-details.html]: done

File: registration/templates/company-profile-details.html (hunk 2 above)
Comment (richtier) [inline registration/templates/company-profile-details.html]: done

**Comment by monikasulik** [comment]
:sunny:

**Pull request merged**